test/protobuf/protoc/proto_gen/extension2.pb.ex

```diff
@@ -93,6 +99,11 @@ defmodule Ext.MyMessage do
     "ext.MyMessage"
   end
 
+  def message_options do
+    # credo:disable-for-next-line
+    Elixir.Brex.Elixirpb.MessageOptions.decode(<<8, 1>>)
```

Comment:
like it would be a bit nicer if this just listed out the message options instead of this  decode call.